lib/raidenclient/RaidenClient.ts

```diff
@@ -99,8 +80,10 @@ class RaidenClient extends BaseClient {
   public address?: string;
   private port: number;
   private host: string;
-  /** Map of token swap identifiers to order ids */
-  private swapIdOrderMap = new Map<number, Order>();
+  private reconnectionTimer?: NodeJS.Timer;
+
+  /** Time in milliseconds between attempts to recheck connectivity to Raiden. */
+  private static readonly RECONNECT_TIMER = 5000;
```

Comment:
This and the `reconnectionTimer` can also be moved to `BaseClient`, I think they'll be used the same way.

Reply:
I'll address this in a future PR as I already have some logic implemented for BaseClient timers (for channelbalance sanity checks/caching).